packages/cloud_firestore/cloud_firestore/example/pubspec.yaml

```diff
@@ -3,32 +3,32 @@ description: Demonstrates how to use the firestore plugin.
 
 environment:
   sdk: ">=2.12.0-0 <3.0.0"
-  flutter: ">=1.12.13+hotfix.4 <2.0.0"
+  flutter: ">=1.9.1+hotfix.5 <2.0.0"
```

Comment:
Any reason this was lowered?

Reply:
Might have been the migration tool @russellwheatley ?